wolfcrypt/src/port/Espressif/esp32_mp.c

```diff
@@ -59,10 +61,11 @@ static int espmp_CryptHwMutexInit = 0;
 */
 static int esp_mp_hw_wait_clean()
 {
-    int timeout = 0;
-    while(++timeout < ESP_RSA_TIMEOUT && DPORT_REG_READ(RSA_CLEAN_REG) != 1){}
+    word32 timeout = 0;
+    while(!ESP_TIMEOUT(++timeout) && 
+                DPORT_REG_READ(RSA_CLEAN_REG) != 1) { }
 
-    if(timeout >= ESP_RSA_TIMEOUT) {
+    if(ESP_TIMEOUT(timeout)) {
         ESP_LOGE(TAG, "waiting hw ready is time-outed.");
```

Comment:
Minor grammer correction - I would change "time-outed." to "timed out."

Reply:
Thanks, fixed.